snap/local/hooks/helper-go.go

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (C) 2022 Canonical Ltd
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ * SPDX-License-Identifier: Apache-2.0'
+ */
+
+package main
+
+import (
+	"os"
+)
+
+func main() {
+	subCommand := os.Args[1]
+	switch subCommand {
+	case "install":
+		install()
+	case "configure":
+		configure()
+	default:
+		panic("Unknown hook sub-command: " + subCommand)
+	}
+}
```

Comment:
This script should be snap/local/helper-go/main.go together with install.go and configure.go.